geomesa-feature/src/main/scala/org/locationtech/geomesa/feature/kryo/SimpleFeatureSerializer.scala

```diff
@@ -0,0 +1,346 @@
+/*
+ * Copyright 2015 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.feature.kryo
+
+import java.util.{Date, List => jList, Map => jMap, UUID}
+
+import com.esotericsoftware.kryo.io.{Input, Output}
+import com.esotericsoftware.kryo.{Kryo, Serializer}
+import com.vividsolutions.jts.geom.Geometry
+import org.locationtech.geomesa.utils.text.WKBUtils
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+import scala.collection.JavaConversions._
+import scala.ref.SoftReference
+
+/**
+ * Kryo serialization implementation for simple features. This class shouldn't be used directly -
+ * see @KryoFeatureSerializer
+ *
+ * @param sft
+ */
+class SimpleFeatureSerializer(sft: SimpleFeatureType) extends Serializer[SimpleFeature] {
```

Comment:
I was trying to avoid ambiguity between this and the KryoFeatureSerializer... Also, this can serialize any simple feature, not just a kryo one (although it deserializes to a kryo impl). Since it's not used directly anywhere except in the KryoFeatureSerializer, I would prefer to keep it as is.